web/jobs/jobs.go

```diff
@@ -58,12 +88,45 @@ func pushJob(c echo.Context) error {
 		return wrapJobsError(err)
 	}
 
-	return jsonapi.Data(c, http.StatusAccepted, &apiJob{job}, nil)
+	accept := c.Request().Header.Get("Accept")
+	if accept != typeTextEventStream {
+		return jsonapi.Data(c, http.StatusAccepted, &apiJob{job}, nil)
+	}
+
+	w := c.Response().Writer
+	w.Header().Set("Content-Type", typeTextEventStream)
+	w.WriteHeader(200)
+	if err := streamJob(job, w); err != nil {
+		return nil
```

Comment:
[not sure] I am not sure how go channels work but wont an error on job serialization leave the connection indefinitively open and block the channnel?